action.yml

```diff
@@ -84,7 +84,9 @@ runs:
 
     - name: Version bump
       id: version-bump
-      if: ${{ github.event_name == 'workflow_dispatch' }}
+      if: |
+        github.event_name == 'workflow_dispatch' &&
+        inputs.semver != 'auto'
```

Comment:
I stopped reviewing a this point. Unless you think there's a good reason to do it the way you did, I don't think we should be changing the approach so much. I mean this step is bumping the package versions and the logic of the action assumes that it's bumped here and that there's a PACKAGE_VERSION output variable populated. If we move this logic elsewhere in case of an auto bump, we may inadvertently break invariants and assumptions that we're making elsewhere.

Now, I do understand that the auto logic is different, but why don't we execute it here instead of elsewhere? Basically, turn this step into: if auto { do something } else { do something else }, then the rest of the action remains unchanged

Reply:
I agree, that would be cleaner approach. I will push an update soon